### PR TITLE
Add OIDC examples for both PKCE and private_key_jwt

### PR DESCRIPTION
**Pull request by zachmargolis**
Also add a fancy picker!

picker

### CODE REVIEW

File: _pages/openid-connect.md [hunk 1]
```diff
@@ -271,3 +340,32 @@ login.gov supports some of the [standard claims from OpenID Connect 1.0][standar
 
  * **sub**
    Unique ID for this user. This ID is unique per client.
+
+<script type="text/javascript">
```
Comment (pkarman) [inline _pages/openid-connect.md]: time for jquery or its ilk?

File: _pages/openid-connect.md [hunk 2]
```diff
@@ -52,18 +69,34 @@ Users need to authorize OpenID Connect 1.0 clients individually. To present the
 
 ### Authorization Request
 
-```
+View example as <button data-example="pkce">PKCE</button><button data-example="private_key_jwt">private_key_jwt</button>
```
Comment (pkarman) [inline _pages/openid-connect.md]: Do we need to define some acronyms? Or are they already defined elsewhere in this doc? "PKCE" and "JWT" etc.

Comment (zachmargolis) [inline _pages/openid-connect.md]: The first few times I use JWT, I link to the JWT spec?

File: _pages/openid-connect.md (hunk 1 above)
Comment (zachmargolis) [inline _pages/openid-connect.md]: This is simple enough, I think it's easier than adding a dependency on jQuery, unless you feel strongly? `document.querySelectorAll`, a new standard, is inspired by jQuery?

Comment (pkarman) [inline _pages/openid-connect.md]: no strong feelings. Just seeing that much raw JS made me wonder.

File: _pages/openid-connect.md (hunk 2 above)
Comment (zachmargolis) [inline _pages/openid-connect.md]: I added a link to the PKCE spec as well

**Pull request merged**